Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3662,6 +3662,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         case CSSPropertyAnimationPlayState:
         case CSSPropertyAnimationTimingFunction:
         case CSSPropertyAppearance:
+        case CSSPropertyBlockStepSize:
```

Comment:
Ah I see, the spec doesn't define whether the property is animatable. Can you file an issue on w3c/csswg-drafts?

Feel free to ignore my comment about animation tests for now.

Reply:
https://drafts.csswg.org/css-rhythm/#block-step-size